pygount/common.py

```diff
@@ -185,3 +187,27 @@ def new_func2(*args, **kwargs):
         return new_func2
     else:
         raise TypeError(repr(type(reason)))
+
+
+if sys.version_info.minor < 7:
```

Comment:
Also check for major version 3.

Maybe something like
```python
if sys.version_info[:2] <= (3, 6): ...
```

Or maybe handle it at the import?
```python
try:
    from contextlib import nullcontext
except ImportError:
   # Fallback for Python 3.6.
   class nullcontext(...):
       ...
```